### PR TITLE
Update release artifact location

### PR DESCRIPTION
**Pull request by trisberg**
Link to the getting-started-guide for install instructions.

Also install kafka with helm instead of our release yaml that is used for internal testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,19 +100,13 @@ cd riff-shopping-demo
...
-./riff-kapp-install.sh
-```
+1. Install the Core runtime by following instructions in the [Core Runtime Install section](https://projectriff.io/docs/latest/runtimes/core#install).
 
-For a cluster like "Minikube" or "Docker Desktop" that doesn't support LoadBalancer use:
-
-```
-./riff-kapp-install.sh --node-port
-```
+1. Install the Streaming runtime by following instructions in the [Streaming Runtime Install section](https://projectriff.io/docs/latest/runtimes/streaming#install).
```
Comment (fbiville) [inline README.md:109]: the instructions for Knative are missing, is that intentional?

File: README.md [hunk 2]
```diff
@@ -100,19 +100,13 @@ cd riff-shopping-demo
...
-Install riff and all dependent packages including cert-manager, kpack, keda, kafka, riff-build, istio and core, knative and streaming runtimes.
+Install riff with Core, Knative and Streaming runtimes plus their dependencies.
 
-For a cluster that supports LoadBalancer use:
+1. Follow the instructions for your Kubernetes installation from the [riff getting started guides](https://projectriff.io/docs/latest/getting-started).
 
-```
-./riff-kapp-install.sh
-```
+1. Install the Core runtime by following instructions in the [Core Runtime Install section](https://projectriff.io/docs/latest/runtimes/core#install).
```
Comment (fbiville) [inline README.md:107]: are we moving away from `riff-kapp-install.sh`?

**Review by fbiville — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (trisberg) [reply README.md:109]: Knative is installed as part of the getting started doc - https://projectriff.io/docs/latest/getting-started/gke#install-riff-knative-runtime (we should remove the core once we switch demo to use knative deployers - https://github.com/projectriff-demo/demo/pull/27)

File: README.md (hunk 2 above)
Comment (trisberg) [reply README.md:107]: No point maintaining separate install script once this demo is promoted as a general reference app for riff, I'd like to change the doc to not require a clone of the repo, just copy-n-paste.

File: README.md (hunk 1 above)
Comment (fbiville) [reply README.md:109]: I don't see this link in the previous paragraph, yet this section starts with `Install riff with Core, Knative and Streaming runtimes plus their dependencies.`.

Comment (trisberg) [reply README.md:109]: The first item in the list of installs has link to install instructions that include  `riff build` and `riff knative runtime`

Comment (fbiville) [reply README.md:109]: gotcha

**Review by fbiville — APPROVED** [review]

**Pull request merged**